src/main/java/edu/tamu/scholars/middleware/discovery/model/repo/impl/IndividualRepoImpl.java

```diff
@@ -264,18 +263,17 @@ private SimpleQuery buildSimpleQuery(List<FilterArg> filters) {
 
     private List<SimpleFilterQuery> buildFilterQueries(List<FilterArg> filters) {
         List<SimpleFilterQuery> results = new ArrayList<SimpleFilterQuery>();
```

Comment:
I'm noticing that the design change is not to return results but use the existing array and append to it.

Would it be better to make the results list `final` given the implemented approach?
```suggestion
        final List<SimpleFilterQuery> results = new ArrayList<SimpleFilterQuery>();
```